packages/cadl-ranch-specs/http/models/visibility/mockapi.ts

```diff
@@ -0,0 +1,66 @@
+import { passOnSuccess, mockapi, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+
+function genData(keys: string[]): Record<string, any> {
+  const ret: Record<string, any> = {}
+  const fullData: Record<string, any> = {
+    readProp: "abc",
+    queryProp: 123,
+    createProp: ["foo", "bar"],
+    updateProp: [1, 2],
+    deleteProp: true,
+  };
+  for (const k of keys) {
+    if (k in fullData) {
+      ret[k] = fullData[k];
+    }
+  }
+  return ret;
+}
+
+Scenarios.Models_Visibility_getModel = passOnSuccess(
```

Comment:
Need to update the mock apis as well to include `_Automatic` in scenario name

Reply:
my bad :( for even missing the pipeline errors. 
I updated to get fixed.